airflow/www/views.py

```diff
@@ -2078,6 +2078,71 @@ def tree(self):
             external_log_name=external_log_name,
         )
 
+    @expose('/calendar')
+    @auth.has_access(
+        [
+            (permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG),
+            (permissions.ACTION_CAN_READ, permissions.RESOURCE_TASK_INSTANCE),
+            (permissions.ACTION_CAN_READ, permissions.RESOURCE_TASK_LOG),
```

Comment:
```suggestion
```

I don't think this makes sense to include, as it doesn't directly read the task logs.